libs/health-insurance/src/lib/soap/healthInsuranceAPI.ts

```diff
@@ -101,7 +106,10 @@ export class HealthInsuranceAPI {
         const filename = arrAttachments[i]
```

Comment:
Should `attachmentsFileNames` be absolute urls to the files on AWS? You can get the URL and the original filename from `application.attachments`.

Reply:
I'm using original filename now.
Will have to get url and use it later when refactor Bucket codes

Reply:
We actually rename the file before storing it on S3 (adding a GUID prefix to deal with duplicate filenames). The real original file name is under `application.attachments `. But maybe you prefer a unique filename with guid?